ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -1,15 +1,9 @@
 from typing import List
 
 from ee.clickhouse.queries.funnels.base import ClickhouseFunnelBase
-from ee.clickhouse.sql.funnels.funnel import FUNNEL_SQL
 
 
 class ClickhouseFunnel(ClickhouseFunnelBase):
```

Comment:
I think this is a good time to clean up `FunnelBase` as well, specially with `get_query` and removing the `format_properties` parameter?

Reply:
oh, and since we have good tests now: can resolve the comma inconsistencies in the base functions here too? (i.e. no private function of base should return query parts with a comma at the beginning).

Although this bit can be done in a separate PR too.

Reply:
And, replacing `format_results`, too! (which removes the TODO here: https://github.com/PostHog/posthog/pull/4943/files#diff-dcd0ba229a9d3211e51b8f5e45b42841ad88b5dc8a2d39c3b65de5af3b77bea7R81

Reply:
I'd make any refactoring changes in a separate PR. I think it's worth testing this for functionality + stability and merging so core experience has access to the features (conversion times, breakdown, ordering) to implement.